core/modules/field_ui/field_ui.admin.inc

```diff
@@ -2333,7 +2333,11 @@ function field_ui_view_mode_delete_form_submit($form, &$form_state) {
  * Access callback: Checks the users access and token to enable a display mode.
  */
 function field_ui_view_mode_enable_access() {
-  return user_access('administer view modes') && backdrop_valid_token($_GET['token'], 'view_mode_enable');
+  $token_check = TRUE;
```

Comment:
default should be FALSE, isn't it ?

Reply:
I don't know. Do we deny access if `$_GET['token']` isn't set? I was guessing we would only deny access if a token WAS set but it wasn't valid.